redis_task/consumer.py

```diff
@@ -0,0 +1,108 @@
+import redis
+import time
+import json
+from db import DatabaseManager
+from dotenv import load_dotenv
+from os import getenv
+from redis.commands.json.path import Path
+
+
+load_dotenv('env')
+
```

Comment:
Add one file (config.py)
Fetch all environment variables in config.py

and import file specific environment variables into your file from config.py

Eg:
from config import DB_NAME, REDIS_PORT

Reply:
Made the changes by creating a new config.py file